crates/uv/tests/lock.rs

```diff
@@ -3902,7 +3983,7 @@ fn lock_requires_python_pre() -> Result<()> {
         assert_snapshot!(
             lock, @r###"
         version = 1
-        requires-python = ">=3.11"
+        requires-python = ">=3.11b1"
```

Comment:
We now retain this more-or-less verbatim. I think this is ok since we'll ignore these anywhere that it matters when we actually _use_ it.